chart/values.yaml

```diff
@@ -667,3 +670,18 @@ obs:
         http: 90011
         # NodePort associated with https port
         https: 90010
+
+localpv-provisioner:
+  # -- Enables the openebs dynamic-localpv provisioner. If disabled, modify etcd and loki-stack storage class accordingly.
+  enabled: true
+  # Enable/disable the openebs NDM sub-chart. It's recommended to keep this disabled.
+  openebsNDM:
+    enabled: false
+  # Enable/disable the creation of the openebs-device StorageClass. It's recommended to keep this disabled.
+  deviceClass:
+    enabled: false
+  hostpathClass:
+    # -- Name of the default hostpath StorageClass. If modified, also modify etcd and loki-stack default storage class accordingly.
+    name: "openebs-localpv-hostpath"
```

Comment:
I think at least contain  mayastor, to avoid confusion, this is specifically for the mayastor chart.

Reply:
Any suggestion. Inserting mayastor would make it very long.`mayastor-openebs-localpv-hostpath`. 
I am trying to avoid `mayastor-localpv-hostpath` because it may again raise questions of mayastor having localpv variant.

Reply:
How about naming what it is for? so  something like `mayastor-etcd-storage`

Reply:
@blaisedias This storage class would be used for loki as well. So we need a probably generic name.

Reply:
@avishnu Suggested we should have `localpv` in the name as well. So `ms-internal-localpv-hostpath` or `ms-config-localpv-hostpath`?